modal/functions.py

```diff
@@ -695,15 +695,22 @@ def from_args(
                 )
                 image = image.extend(build_function=snapshot_function, force_build=image.force_build)
 
+        if keep_warm is not None:
+            assert isinstance(keep_warm, int)
```

Comment:
I know you're just moving this around but it caught my eye: for user-facing errors I think it's preferable to do something like `raise TypeError(f"keep_warm must be an int, not {type(keep_warm).__name__}"`.

Reply:
Done.